components/advanced_create_post/advanced_create_post.tsx

```diff
@@ -1494,13 +1497,25 @@ class AdvancedCreatePost extends React.PureComponent<Props, State> {
         this.focusTextbox();
     };
 
+    handleFocus = () => {
+        this.setState({showPostPriorityTourPoint: true});
```

Comment:
It seems a bit odd to me that this always tries to show the tour point on focus and then the component itself determines whether or not it should actually be visible when that happens. That'd be fine if it appeared every time the user focused on the textbox, but it seems weird because the tooltip isn't always visible when `showPostPriorityTourPoint` is true. I'm thinking of the following case:

1. It starts as `false`.
2. When the user focuses the textbox, it goes to `true`, and the tour tip appears.
3. It goes to `false` when the user dismisses the tour tip and then it disappears.
4. Next time the user focuses the textbox, it goes back to `true`, but the tour tip doesn't reappear because it's hidden by preferences.

I'm trying to think if there's a way to make that work better without having this component needing to know more about the preference that controls the tooltip reappearing.

Perhaps we could instead change this so that this is replaced by a generic `focused` state that goes true/false as the input is focused/blurred normally, and then `PostPriorityTourTip` can set its `showTip` state based on the preference when it sees `focused` go from false to true? That'd also mean that we don't need this component to worry about handling the `success` case since it just requires one `focused` prop

Reply:
+1 on the last part of your comment, I think the component only needs to know when it's in focus or blurred, so passing a prop to the tour tip component and handling the success case there would be cleaner.